collector.go

```diff
@@ -16,9 +16,15 @@ import (
 )
 
 var (
-	workflowRunTimeVec = prometheus.NewCounterVec(
+	workflowRunRunnerTimeVec = prometheus.NewCounterVec(
```

Comment:
"Time" should be removed from this variable name - it was matching the metric name.

But a number of the var names should be updated to match the metric names, particularly given my suggestion to s/run_time/runner/ throughout.